src/docs/environment/index.mdx

```diff
@@ -2,7 +2,11 @@
 title: "Environment"
 ---
 
-This guide steps you through configuring a local development environment for the Sentry server on macOS. If you're using another operating system the instructions are still roughly the same, but we don't maintain any official documentation for anything else for now.
+*NOTE*: There's some documented known issues at the bottom of this page.
```

Comment:
Are you referring to the troubleshooting section here? If so, best to just add a link. And move it to the end of the same intro paragraph, the link is enough for a visual distinguishment vs. _NOTE_

Reply:
Done.